### PR TITLE
Get fixed table headers working.

### PR DESCRIPTION
**Pull request by andylolz**
It looks like floatThead was partially added, but I’m not sure it was ever working (and instead just throws javascript errors).

This pull request gets fixed table headers working. See:
ro1z05bjnw

I’m not sure I’m sold on it tbh! but I suppose it should either be fully added or fully removed.

### CODE REVIEW

**Comment by andylolz** [comment]
Blocked until #543 is merged.

File: extra/html/data_table.html [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
-    scrollContainer: function($table) {
-      return $table.closest('.table-container');
-    }
-  });
-})
-</script>
-
-<script type='text/javascript' src='{{ site.github.url }}/js/papaparse.min.js'></script>
-<script>
+$(function() {
```
Comment (quadrophobiac) [inline extra/html/data_table.html]: a question purely for my own knowledge - is this way preferred to using document.ready?

Comment (quadrophobiac) [inline extra/html/data_table.html]: @andylolz

Comment (andylolz) [inline extra/html/data_table.html]: It’s the one I always use. [According to this](http://learn.jquery.com/using-jquery-core/document-ready/), apparently that makes me an ‘experienced developer’!

**Pull request merged**